src/pages/index.mdx

```diff
@@ -4,23 +4,69 @@ import { SiteLayout } from "../components/layout/SiteLayout";
 
 <SiteLayout>
 
-<h1 className="text-center font-display">11th Annual oSTEM Conference 2021</h1>
-
-<ConferenceCountdown />
-
-<img src="/images/2021/podium-presenters.jpg" alt="" className="rounded-3xl p-3"/>
-
-<p className="text-center text-3xl font-display">Your seat at our table</p>
-<p className="text-center text-3xl font-display">
-  October 28 &ndash; 31, 2021, Virtual
-</p>
-
-<div className="text-2xl flex justify-center p-8">
-  <CTAButton onClick="/2021/rfp">Submit a Proposal</CTAButton>
+<div className="border-b-4 border-mulberry border-solid pb-4">
+    <p className="flex text-center text-5xl font-display font-black text-mulberry">11th Annual oSTEM Conference 2021</p>
+    <p className="flex justify-center text-2xl font-display italic">Your seat at our table</p>
+    <hr/>
+    <p className="flex justify-center text-3xl font-display">
+    October 28 &ndash; 31, 2021 | Virtual
+    </p>
+</div>
+<div class="grid grid-cols-5 gap-y-2 mt-9">
+    <div class="rounded-l col-span-2 tint tint-dark"><img class="object-cover w-full h-full rounded-l" src="/images/oSTEM.2019.0049_Update.png"/></div>
```

Comment:
I did try to use next/image previously, but I could not get the image to show up during local test development. I tried hunting down the specific error but ultimately it wasn't fruitful. Because the original image did not use the `next/image` component, I kept in that same style and used a basic `<img>` tag.

Reply:
The first image is 22mb, so using just an `<img />` tag is a non-starter.

This works for me:
```html
<div className="col-span-2 tint tint-dark rounded-l overflow-hidden">
    <Image
      src="/images/oSTEM.2019.0049_Update.png"
      layout="fill"
      objectFit="cover"
      alt=""
    />
  </div>
  <div className="col-span-3 bg-tint-spruce-light p-4 rounded-r overflow-hidden">
    <h3>Conference for LGBTQ+ people in STEM Fields</h3>
    <p>
      The oSTEM Conference returns agains this year in a virtual format. This
      conference is a space for LGBTQ+ students and professionals in STEM fields
      to share their experiences and learn from one another. We have programs
      ranging from safe spaces to talk about queer issues to discussing the
      latest in research conducted by our members.
    </p>
    <p>
      If you want a space where you can be your authentic LGBTQ+ self and your
      authentic STEM self, this is the conference for you.
    </p>
    <div className="text-center my-4">
      <CTAButton onClick="https://events.bizzabo.com/343425">
        Register for the Conference
      </CTAButton>
    </div>
  </div>
```

As an aside, the images don't have alt tags. They should definitely be empty here, but they're still required for accessibility (`alt=""`).

Reply:
So, actually, I was wondering why that image is 23MB... and I realized it's because it's a PNG of a photo. Do you have the JPEG version to use here instead?

Reply:
So using that code (which is what I tried previously and couldn't chase down why it wasn't working), this is what I get locally
![image](https://user-images.githubusercontent.com/20641196/129613236-a9417e7f-37fb-4b4d-8e1c-1e51f1b5e1eb.png)

Reply:
The image is a png because it has editing done to obscure faces for privacy reasons. We can't use the original image as-is.

Reply:
Can we export the edited file as a JPEG? JPEG's have much better compression than PNGs (it's lossy, but doesn't usually matter in the context of very detailed things like photos).

It works for me™. Maybe an obvious question, but did you `import Image from "next/image"` at the top of the file?

Reply:
>Can we export the edited file as a JPEG? 

Probably not since I don't think we have the raw files.

Reply:
I did make sure to import it, maybe it's something weird the styling? Looking into next/image and styling I am less than impressed by the workarounds you have to use. Either way, for the time being, I've found a smaller image for that first photo. I'm using that and will continue to investigate why next/image is disliking me.